pkg/apis/numaflow/v1alpha1/udf.go

```diff
@@ -122,13 +122,21 @@ type GroupBy struct {
 type Window struct {
 	// +optional
 	Fixed *FixedWindow `json:"fixed" protobuf:"bytes,1,opt,name=fixed"`
+	// +optional
+	Sliding *SlidingWindow `json:"sliding" protobuf:"bytes,1,opt,name=sliding"`
```

Comment:
it should be `2`

Reply:
yes did not intend to push this originally. pushed the changes now.